backend/src/api.ts

```diff
@@ -123,6 +124,12 @@ const loginCheckedDelete = (
   handler: (req: Request, user: IdolMember) => Promise<Record<string, unknown>>
 ) => router.delete(path, loginCheckedHandler(handler));
 
+// Email notifs
+router.post('/sendMail', async (req, res) => {
+  sendMail(req.body.to, req.body.subject, req.body.text);
+  res.status(200);
+});
```

Comment:
I don't think we necessarily need this to be an endpoint. For something that needs an email notification like `/updateMember`, we can probably just call `sendMail()` directly from `updateMember()` in `membersAPI.ts`.

Reply:
Sounds good! yeah, I was initially going to draft the email and call this in front end, but I see how it is just easier to do it all in backend and avoids unnecessary calls :)

Reply:
Actually it might not be a bad idea to leave this in there for now just for testing purposes... sorry lol

Reply:
np, just reverted this